### DISCUSSION TITLE
toitpkg: pkg subcommand

### DISCUSSION THREAD

**Discussion by rsteube** [toitlang/toit#31] [Ideas]
Are more subcommands planned for toitpkg? Since it is centered around package management it seems to me the pkg subcommands might as well be ordered right under the root command.

**Comment by floitsch** [comment]
Good question :)

The reason it's currently in a subcommand is, that it was designed for the `toit` CLI where it just provides a subcommand.
I do agree that the additional subcommand is now a bit redundant and awkward. I thought of removing it, but there are other things that are currently more important so I didn't spend time on it. It should be pretty safe to do the change later.
